.tekton/operator-bundle-build.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: tekton.dev/v1
```

Comment:
Once you diff this file with any other `.tekton/*-build.yaml`, you'll see discrepancies. For the most part they shouldn't exist.